app/views/forms/incident-report/_request-info.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= f.input :type_of_incident, as: :radio_buttons, collection: ["Vandalism", "Accident/Medical", "Theft", "Harassment/Fight/Abusive Language", "Building Issue", "Other"], required: :true %>
+<%= f.input :other_incident, label: "Other Incident Type", as: :text, required: :false, size: "60", maxlength: "128" %>
+<%= f.input :time_of_incident, label: "Time of Incident", as: :time, required: :true %>
+<%= f.input :date_of_incident, label: "Date of Incident", as: :date, required: :true %>
+<%= f.input :location_where_incident_occured, label: "Location Where Incident Occured", as: :text, required: :true %>
+<fieldset class="webform-component-fieldset form-wrapper" id="webform-component-primary-relevant-person"><legend><span class="fieldset-legend">Primary Relevant Person, if needed</span></legend><div class="fieldset-wrapper"><div class="form-item webform-component webform-component-textfield" id="webform-component-primary-relevant-person--identify-relevant-persons">
+<%= f.input :primary_relevant_person, label: "Primary Relevant Person", as: :text, required: :true %>
+<%= f.input :primary_email, label: "Provide Email, if needed", as: :text, required: :false %>
+<%= f.input :primary_affiliation, label: "Affiliation", as: :radio_buttons, collection: ["TU Student", "TU Faculty or Staff", "Library Student Assistant", "Library Staff", "Visitor/Guest"], required: :true %>
```

Comment:
We've been trying to find re-usable collections that appear throughout our forms. I've been adding them as collections in the application.rb file rather than inline as these are here. Could you compare what we have with what you need and re-use or create as appropriate?

Reply:
Moved collections into config/application.rb